e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -1546,3 +1548,46 @@ describe("issue 49321", () => {
     });
   });
 });
+
+describe("issue 44665", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+  });
+
+  it("should use the correct widget for the default value picker (metabase#44665)", () => {
+    openNativeEditor().type("select * from {{param");
+    sidebar()
+      .last()
+      .within(() => {
+        cy.findByText("Search box").click();
+        cy.findByText("Edit").click();
+      });
+
+    modal().within(() => {
+      cy.findByText("Custom list").click();
+      cy.findByRole("textbox").type("foo\nbar\nbaz");
+      cy.button("Done").click();
+    });
+
+    sidebar().last().findByText("Enter a default value…").click();
+    popover()
+      .findByPlaceholderText("Enter a default value…")
+      .should("be.visible");
```

Comment:
Let's also assert that filter values are shown in the popover

```suggestion
    popover().within(() => {
      cy.findByPlaceholderText("Enter a default value…").should("be.visible");
      cy.findByText("foo").should("be.visible");
      cy.findByText("bar").should("be.visible");
      cy.findByText("baz").should("be.visible");
    });
```